packages/atomic/src/components/search/atomic-layout/atomic-search-layout.pcss

```diff
@@ -29,7 +29,7 @@ atomic-search-layout {
   grid-template-areas:
     '. atomic-section-search .'
     '. atomic-section-main .';
-  grid-template-columns: var(--atomic-layout-spacing-x) minmax(18rem, 1fr) var(--atomic-layout-spacing-x);
+  grid-template-columns: var(--atomic-layout-spacing-x) minmax(0, 1fr) var(--atomic-layout-spacing-x);
```

Comment:
Not sure why there was a minimum width here. Any clue?

Reply:
Last change on this line was about fixing things for table layout 🤔